tools/test/generateAllScreenshots.py

```diff
@@ -64,13 +64,14 @@ def deleteDuplicatedScreenshots(lang):
     print("Deleting screenshots identical to the English version for language %s..." % lang)
     files = os.listdir("tests/uitests/src/test/snapshots/images/")
     # Filter files by language
-    files = [file for file in files if file[-7:-5] == lang]
+    files = [file for file in files if file[-6:-4] == lang]
     identicalFileCounter = 0
     differentFileCounter = 0
     for file in files:
-        englishFile = file[:3] + "S" + file[4:-7] + "en" + file[-5:]
+        englishFile = file[:-6] + "en" + file[-4:]
         fullFile = "tests/uitests/src/test/snapshots/images/" + file
         fullEnglishFile = "tests/uitests/src/test/snapshots/images/" + englishFile
+        print("File: %s, enligshFile: %s" % (fullFile, fullEnglishFile))
```

Comment:
Maybe comment the `print`

Reply:
Ooops, I was sure I had remove these. Thanks.